### DISCUSSION TITLE
Is there any plugins available  inorder to have rulers inside grapejs?

### DISCUSSION THREAD

**Discussion by Ananthumpillai** [GrapesJS/grapesjs#5722] [Q&A] [answered]
I have already tried this one [grapesjs-rulers](https://github.com/Ju99ernaut/grapesjs-rulers), its actually good.
The issue that I am facing with this is, when I click on the button for the second time(to hide the rulers) the dragmode(dmode) is set to default. I don't want that behaviour. I have already set dragmode as translate in my editor, so on hiding the rulers it is being changed.

**Comment by artf** [comment, marked as answer]
Maybe you can check the code of the plugin and see how to change it 🤷‍♂️

**Reply by Ananthumpillai** [reply]
Thanks, I made a minor change in the plugin and now its working
